### PR TITLE
stop scripts on any error

### PR DESCRIPTION
**Pull request by mhamrle**

### CODE REVIEW

File: golang-base/install.sh [hunk 1]
```diff
@@ -1,4 +1,5 @@
 #!/bin/sh
+set -e
```
Comment (benma) [inline golang-base/install.sh]: Needs a comment.

Comment (toxeus) [inline golang-base/install.sh]: I prefer 
```bash
#!/bin/sh -e
```

**Comment by benma** [comment]
:+1:

**Comment by toxeus** [comment]
:+1:

**Pull request merged**